### DISCUSSION TITLE
Reset footer

### DISCUSSION THREAD

**Discussion by aheil** [marp-team/marp#56] [Q&A] [answered]
When using the `footer` directive e.g. `<!-- footer: foo -->` in a particular slide, it is not possible to reset/remove the footer anymore. 

Actually, this means: If no global footer is used,  all slides up to the point where the footer is introduced do not provide any footer. From that point on the footer exists on each following page. 

It can be changed, whoever, it cannot be removed anymore. 

While this behavior might be exactly as designed, the possibility to reset the footer would be great.

**Comment by yhatt** [comment, marked as answer]
Set an empty string with quotes to remove footer/header.

```
<!-- footer: "" -->
```

**Reply by aheil** [reply]
Thanks a lot. I tried various ways using white spaces, empty footer directive, however I did not try the obvious.

**Reply by ehaselwanter** [reply]
is it possible to have that only for the one slide you want to switch it of? with this one needs to add the foot on the next slide again with the footer directive ....

**Reply by yhatt** [reply]
@ehaselwanter Try [scoped local directive (spot directive)](https://marpit.marp.app/directives?id=apply-to-a-single-page-spot-directives). Just add underscore `_` to the head of the local directive name: `<!-- _header: "" -->`.

**Reply by ehaselwanter** [reply]
@yhatt that does the trick. thx :-)
